ElectronClient/gui/MainScreen.jsx

```diff
@@ -238,6 +238,7 @@ class MainScreenComponent extends React.Component {
 							if (answer !== null) {
 								try {
 									tag.title = answer;
+									tag.rename = true;
```

Comment:
Tags don't have a "rename" property.

Reply:
I actually add one to distinguish whether it is a rename operation or add operation because parameters are always same in both the operation